### PR TITLE
add thread name

### PR DESCRIPTION
**Pull request by P4Cu**
Simple change to show thread name.

### CODE REVIEW

**Comment by cs01** [comment]
Thanks! Will test then merge when I find the time to

File: gdbgui/src/js/Threads.jsx [hunk 1]
```diff
@@ -115,7 +115,7 @@ class Threads extends React.Component {
     }
     return (
       <span key={'thread' + thread.id} className={`${cls}`} style={{fontSize: '0.9em'}}>
-        {status} {Memory.make_addrs_into_links_react(thread['target-id'])}, core {thread.core}, {thread.state}, id {thread.id}
+        {status} {Memory.make_addrs_into_links_react(thread['target-id'])}, core {thread.core}, {thread.state}, id {thread.id}, name {thread.name}
```
Comment (cs01) [inline gdbgui/src/js/Threads.jsx]: I am looking at some sample output and don't see a "name" field on the thread. Which version of gdb are you running?

Comment (P4Cu) [reply gdbgui/src/js/Threads.jsx]: GNU gdb (GDB) 8.1
maybe it should be an optional then

**Comment by P4Cu** [comment]
I've made the thread.name optional as you've asked :-)

File: gdbgui/src/js/Threads.jsx [hunk 2]
```diff
@@ -133,7 +133,7 @@ class Threads extends React.Component {
     return (
       <span key={'thread' + thread.id} className={`${cls}`} style={{fontSize: '0.9em'}}>
         {status} {Memory.make_addrs_into_links_react(thread['target-id'])}, core{' '}
-        {thread.core}, {thread.state}, id {thread.id}
+        {thread.core}, {thread.state}, id {thread.id}, {thread.name && 'name'} {thread.name}
```
Comment (cs01) [inline gdbgui/src/js/Threads.jsx]: recommend 
```
{thread.name ? `name ${thread.name}` : ''}
```

Comment (P4Cu) [reply gdbgui/src/js/Threads.jsx]: ok, unfortunately JS is not my world :-)

**Pull request closed without merging**